### ISSUE TITLE
Questions about the Implementation and Time to release model.

### ISSUE THREAD

**Issue by Codefmeister** [dong03/MVSS-Net#1]
Hello!
Thanks for your inspiring work. And I 'm very intersted in it.
Last two weeks, I read your great work and decided to re-Implement your network according to the paper.
However, after two weeks, the network is roughly finished, and I trained it in the CASIA_v2 dataset. But the performance is not so good. I thought there must be something wrong with my code. So I have a few questions here.
1. The red arrow in the Network structure fig means upsample by bilinear interpolate? It is a little confusing as the legend not mentioned it.
2. What is the epochs and scheduler of the training process? Following your paper, I set the epoch to 75, and [20, 35, 50] the learing rate will multiple 10^-1.
3. Questions about the loss. When it comes to authenic img, the loss only consists class-supervision, will this loss be multiplied by 0.04?
4. The network structure implemented by myself may contain other questions, so when will you release the model?  I'm very excited and appreciated if you could opensource the model.
Thanks for your patience to read my questions. Sinerely hope that your research could comes to next level.

**Comment by dong03** [comment]
Thanks for your interest.
For implement details:
1. Indeed it is bilinear upsampling, and we will fix the missing in the paper.
2. In our experience, the training process may not exist that long when an early stop strategy is adopted. The decayed trick of learning rate in mvssnet is like (not precise):
![Image text](https://raw.githubusercontent.com/dong03/picture/main/lr.png)
For every epoch lr decays from the top to the bottom(1e-7), and the peak also decays. So your lr may be a little bit higher.
3. Yes, it will be.  
4. Again, thanks for your attention. Currently the test code and model pretrained on CASIAv2 are under internal review, and we will release them once they are ready.

Best regards

**Comment by Codefmeister** [comment]
Thanks for your patient and clear answer, I will try to fix my code according to it. Looking forward to the release. 
Thanks again.
Best regards
